week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -9,6 +9,6 @@ console.log(result); // Expected result 8 - Run the program and make sure it wor
 
 
 // Answer these questions:
-// 1. How many parameters does the function take?
-// 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+// 1. How many parameters does the function take? three
+// 2. What is the function name? add Number 
+// 3. Where is the function called? with what parameters? the function called in line three with papmeters a,b and c
```

Comment:
Actually line 3 is the *definition* of a function. The function is *called* in line 7. The parameters it is called with are 1, 3, and 4. Let me know if you are confused about the difference.